pkgs/applications/virtualization/vmware-workstation/default.nix

```diff
@@ -21,11 +22,15 @@
 , libX11
 , libXi
 , kmod
+, python2 ? enableMacOSGuests
```

Comment:
that does not do what you think: if `python2` is ever removed from nixpkgs, this will replace `python2` by either `true` or `false` depending on the value of `enableMacOSGuests`

Reply:
@symphorien Ah good catch - what would you suggest as a resolution to this?

The patcher script mentions that it requires python2 - personally I think we could probably use python3 - may verify this.